### ISSUE TITLE
[CNTOOLS] Documentation with flow

### ISSUE THREAD

**Issue by rdlrt** [cardano-community/guild-operators#189]
We prolly need to draw up a basic workflow of operations before we share this further.
Markdown format allows flowcharts using mermaid , so perhaps if we could whip a very simple workflow of operations.

Prolly important  to not be too detailed, as this would be part of shorter path for users, but still a basic flow on how to go through the steps wrt cntools (create wallet, upgrade, receive funds, create pool, register pool, delegate) would prolly save a bunch of writing and reading 😄 - from end user's view

**Comment by papacarp** [comment]
# Quickstart for using CNTOOLS
CNTOOLS is a simple shell script that will simplify typical operations used in FF and HTN testnets.  We have developed this for our own use, and as such tend to follow some well worn paths when using it.  See below for simple step by step guide for getting a pool setup on the FF network.  Please note that this tool is tested on linux platforms only at this point.

### Configuration
CNTOOLS connects to your node through the configuration in the env file located in the same directory as the script.  Customize this file for your needs.  CNTOOLS will start even if you node is offline, but don't expect to get very far.

## Start
Insure the tool is executable and start it with:
```
$ ./cntools.sh
```
You should get a screen that looks something like this:
```
 >> CNTOOLS <<                                       A Guild Operators collaboration
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   Main Menu
   1) update
   2) wallet  [ new / upgrade | list | show | remove |
                decrypt / unlock | encrypt / lock ]
   3) funds   [ send | delegate ]
   4) pool    [ new | register | list | show | rotate KES |
                decrypt / unlock | encrypt / lock ]
   q) quit
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
What would you like to do? (1-4):
```
# Step by Step to create a pool with CNTOOLS

1. Choose **wallet** (option #2)
```
 >> WALLET
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   Wallet Management

   1) new / upgrade
   2) list
   3) show
   4) remove
   5) decrypt / unlock
   6) encrypt / lock
   h) home
   q) quit
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
What wallet operation would you like to perform? (1-6):
```
2. Choose **new / upgrade** (option #1)
```
 >> WALLET >> NEW / UPGRADE
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   Wallet Type

   1) payment  - First step for a new wallet
                 A payment wallet can send and receive funds but not delegate/pledge.

   2) stake    - Upgrade existing payment wallet to a stake wallet
                 Make sure there are funds available in payment wallet before upgrade
                 as this is needed to pay for the stake wallet registration fee.
                 A stake wallet is needed to be able to delegate and pledge to a pool.
                 All funds from payment address will be moved to base address.
   h) home
   q) quit
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Choose wallet type (1-2):
```
3. Choose **payment** type of wallet (option #1)
4. Give the wallet a name
5. CNTOOLS will give you your payment address.  For example:
```
>> WALLET >> NEW >> PAYMENT
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Name of new wallet: bob

Wallet: bob
Payment Address: 60ca926192c09b2fc689b5258d9265038c4270e1f1934413255822db1f9645e2c1
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
6.  **Send some money to this wallet.**  Either through the faucet or have a friend send you some.  IMPORTANT...  The wallet must have funds in it before you can proceed.
7.  From the main menu choose **wallet** (option #2) and then this time choose **stake** (option #2) to upgrade your wallet
8.  CNTOOLS will give you a list of wallets you can upgrade.  choose the wallet you created in step 4


SAVING THIS HERE WHILE I SPIN NEW GENESIS

**Comment by papacarp** [comment]
where should this documentation live? readme.md in the scripts section?

**Comment by rdlrt** [comment]
I'd say [here](https://github.com/cardano-community/guild-operators/blob/master/docs/Scripts/cntools.md)

**Comment by gufmar** [comment]
Perfect. I had a plan to do this during weekend. Now it's already done

**Comment by papacarp** [comment]
pr in #197

**Comment by Scitz0** [comment]
Merged into master by 9ce59b1
